packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -164,8 +164,8 @@ const getColorToken = ({
 }): IconColors | TextColors => {
   let state = currentInteraction;
   const map = {
-    default: 'subtle',
-    hover: 'normal',
+    default: 'normal',
+    hover: 'subtle',
```

Comment:
Intentional? Was this a miss while refactoring Link?

Reply:
Yes it's intentional. [Slack Thread](https://razorpay.slack.com/archives/G01B3LQ9H0W/p1704798533721449)